gke/cluster.tf

```diff
@@ -21,10 +21,11 @@ resource "google_container_node_pool" "converge_node_pool" {
 
   management {
     auto_upgrade = true
+    auto_repair  = true
```

Comment:
Any particular reason for adding this?

Reply:
In the event that there arise an issue with a node, it auto repairs.